CHARTER.adoc

```diff
@@ -1,28 +1,41 @@
  . *Mission of the TODO Group (“TODO”).*
  The TODO Group’s mission is to identify key policy and process choices related to corporate open source engagement and create tools and educational materials that promote best practices around such engagements.
  . *Membership*
-The TODO Group membership is composed of individuals deeply involved in running open source programs setup to drive open source in their company. These individuals have learned from practical experience what it takes to successfully engage in or create open source projects and build effective internal open source programs.  +
-The TODO Group shall be composed of the General Members and a Steering Committee. General Members shall be entitled to:
- .. Participate in TODO Group Working Groups, meetings, initiatives, events and any other activities;
- .. identify their company as a member or participant in the TODO Group; and
- .. nominate one individual for election to the Steering Committee.
+The TODO Group membership is composed of individuals deeply involved in running open source programs setup to drive open source across divisions and product lines within their company. These individuals have learned from practical experience what it takes to successfully engage in or create open source projects and build effective internal open source programs that guide how developers in their company engage in external communities. +
+The TODO Group shall be composed of the General Members and a Steering Committee. Anyone may participate as a General Member as long as they meet the General Member Qualifications. At such time as a General Member no longer meets the qualifications, their General Member status will be rescinded.
+ .. General Member Qualifications: An individual may be qualified to join TODO Group as a General Member if they:
+ ... Are an employee of an organization with development teams that use or contribute to open source projects across multiple divisions, products, or departments,
+ ... in a role that manages, oversees, or guides the organization’s policies and contributions to open source projects (an “Open Source Program Office”),
```

Comment:
I support merging these changes, but I think they can be improved on. As written, the preceding two lines could be read to disqualify small companies, new programs, or offices that focus primarily on consumption and compliance. Further, the preamble frames TODO only as a collection of existing experts, and I think we can improve this messaging to frame TODO as the place that open source program officers come to share and develop best practices, regardless of their level of experience.

Reply:
+1 to @DuaneOBrien . A lot of people are very tentative in this space. Fearful of looking dumb or asking goofy questions etc. The wording here feels "elitist" (sigh). While I think I understand the intent etc. it will just take one vocal person reading it at face value and making a twitter storm that we then have deal with. Before merging in think we need to be more inclusive there.

Suggest "aspiring to" or some such around the first couple sentences.

We could also split out this change and get the SC structure part in...